### DISCUSSION TITLE
How to hide local ip from fastfetch

### DISCUSSION THREAD

**Discussion by AnkurAlpha** [fastfetch-cli/fastfetch#629] [Q&A] [answered]
Hi  want to hide my local ip from my fastfetch which currently looks like this : 
![2023-11-22_16-16](https://github.com/fastfetch-cli/fastfetch/assets/132189303/f314f0ed-1a3f-47fa-8e5d-c06865331032)
I don't have special config setup ,
actually there is no config for fastfetch in my system currently,
I just use this : 
~~~
fastfetch --logo-type sixel --logo ~/.config/p.jpg
~~~
in my fish config to get a picture .
Thanks in advance for help :slightly_smiling_face:

**Comment by AnkurAlpha** [comment, marked as answer]
found solution in :  https://github.com/fastfetch-cli/fastfetch/issues/133

**Comment by CarterLi** [comment]
You may generate a config file `fastfetch --gen-config jsonc` and remove `localip` in it

**Reply by AnkurAlpha** [reply]
Thanks for suggestion :-)

**Comment by RobTFirefly** [comment]
Another quick, easy, and novice-friendly way to hide local IP from fastfetch output is to just disconnect from the net for a moment, running fastfetch while offline. There's no IP address if there's no I to P with.
